tests/test_user_manager.py

```diff
@@ -0,0 +1,55 @@
+"""Korone is a simple multipurpose Telegram Bot.
+This is a complete rewrite of PyKorone, refer to
+https://github.com/AmanoTeam/PyKorone and README.md for more
+information."""
+
+import os, sys
+
+sys.path.insert(0, f"{os.getcwd()}/src")   # makes possible to call
```

Comment:
There is no need for this line, korone should be installed on the local environment and then tested with this module.